.github/workflows/scripts/extract-npx-version.sh

```
@@ -0,0 +1,25 @@
+  echo "❌ Invalid tag format '$TAG_NAME'. Expected format: npx/vMAJOR.MINOR.PATCH"
+  exit 1
+fi
+
+# Extract version (remove 'npx/v' prefix to get just the version number)
+VERSION=${TAG_NAME#npx/v}
+
+echo "📦 Extracted NPX version: $VERSION"
+echo "🏷️ Full tag: $TAG_NAME"
+
+# Set outputs
+echo "version=$VERSION" >> $GITHUB_OUTPUT
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)